src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs

```diff
@@ -320,12 +320,8 @@ public void Push()
 			}
 
 			// create a bundle to push
-			string tip = _repo.GetTip().Number.Hash;
-			string bundleId = "";
-			foreach (var revision in baseRevisions)
-			{
-				bundleId += String.Format("{0}-{1}", revision.Number.Hash, tip);
-			}
+			var tip = _repo.GetTip().Number.Hash;
+			var bundleId = string.Join("_", baseRevisions.Select(rev => rev.Number.Hash)) + '-' +  tip; // REVIEW (Hasso) 2017.02: how much of this is necessary? It's too long when used as a filename. (see also line 639)
```

Comment:
Does the revision number have a short version of the hash? If so I think we could use that safely. If not LGTM.